frontend-ssr/redux/exercise-submissions/exercise-submissions.actions.ts

```diff
@@ -0,0 +1,30 @@
+import { Submission } from './types';
+
+const ADD_SUBMISSION = 'exercise-submissions/ADD';
+const REMOVE_SUBMISSION = 'exercise-submissions/REMOVE';
+const UPDATE_SUBMISSION = 'exercise-submissions/UPDATE';
+
+export const EXERCISE_SUBMISSIONS = {
+  ADD_SUBMISSION,
+  REMOVE_SUBMISSION,
+  UPDATE_SUBMISSION,
+};
+
+export const addSubmission = (submission: Submission, index: number) => ({
```

Comment:
The types are wrong. If we can either send a submission or an array, we should have 

```
(data : Submission | Submissions[], index: number)
```

Also, let's add a default value to index.